config/test/bidder-info-invalid/someBidder-invalid.yaml

```diff
@@ -0,0 +1,3 @@
+maintainer:
+  email: 123
+gvlVendorID: "42"
```

Comment:
Why do we need do add this? Unit tests should cover this already.

Reply:
removed. Just wanted to test it with actual yaml file.